### ISSUE TITLE
Bad user experience with MIN_REALIZATIONS

### ISSUE THREAD

**Issue by eivindjahren** [equinor/ert#4328]
The user experience of selecting less realizations (in the active mask field in the gui) than MIN_REALIZATIONS with EnsembleSmoother is really bad. When you start running, it doesn't start up anything (0 / 0 waiting), and displays more than 17000 days left. Only once you close the run_dialog do you get an error message telling you that you selected less than MIN_REALIZATIONS.

**Comment by mortalisk** [comment]
![Screenshot from 2022-11-17 14-42-23](https://user-images.githubusercontent.com/596662/202703530-abbe3071-6a35-47e9-8458-8b1b9685300f.png)

**Comment by sondreso** [comment]
Closing as duplicate of #3727
